1. added 'Temperature VS Relative Humidity' chart in (PMV ASHRAE)/(PMV EN)/Compare; 2. added 'Air speed vs Operature Temperature' chart in PMV ASHRAE

lja996877259 opened the pull request:
The ‘Temperature VS Relative Humidity’ and ‘Air speed vs Operature Temperature’ charts' newest version is merged into the project and will not affect the existing program. Users can select the these charts in the dropdown box, then enter the corresponding parameters. The charts on the right shows the corresponding visualization.
1.Temperature VS Relative Humidity:
- [x]  Added 'Temperature VS Relative Humidity' charts into the project, and combine 'PMV ASHRAE' and 'PMV EN' into one function to simplify the code.
- [x]  In the PMV ASHRAE model, the chart will show the 'Relative Humidity', 'Dry-bulb Temperature' and comfort level.
- [x] In the PMV ASHRAE model, the chart can be compared now.
- [x] In the PMV EN model, the chart will show the 'Relative Humidity', 'Dry-bulb Temperature' and category level with different colors indicating different categories.
- [x] Change the show results of 'Temperature VS Relative Humidity', when temperature outside the comfort area, display prompt information.
- [x] SI&IP units conversion.

2.Air speed vs Operature Temperature:
- [x] Added 'Air speed vs. operative temperature' chart in PMV ASHRAE model into the project.
- [x] SI&IP units conversion.

Unit Test
1.'Temperature VS Relative Humidity' chart in PMV ASHRAE model:
![image](https://github.com/user-attachments/assets/6cc2bd88-7d86-442e-80a2-8bc4b341565c)
 When the red dot outside the neutral sensation region, the corresponding '✘ Does not comply with ASHRAE Standard 55-2023' will be displayed above the show result. 
![image](https://github.com/user-attachments/assets/c04cb0b7-522c-46a4-a860-f1c8b76ae5b2)
 Can manually select SI or IP unit: 
![image](https://github.com/user-attachments/assets/b9ccf754-df39-4d20-ad2d-2fdc1f29344f)
 In addition, I would like to point out a note: when the unit is switched to 'IP', the value of 'tdp' parameter in annotation should also be switched to the value corresponding to the degree Fahrenheit (this value is indeed inconsistent with the value of the original page, I think the original page is wrong). 
![image](https://github.com/user-attachments/assets/6aec32c2-2ab7-4f4c-af66-12513552a5f5)
The codes for PMV EN and PMV ASHRAE will be combined to simplify the code: 
![image](https://github.com/user-attachments/assets/e73e9762-d035-400f-b253-2277911276ab)
In addition, Marcusxixi mentioned that 'with the same input values, both the output results and the chart results are slightly inconsistent when compared to the original website', I think it happens due to the values precision from the data in the input box, and then the data in the final display of chart also has some  discrepancies.
![image](https://github.com/user-attachments/assets/61ea8f9e-a07d-473e-b60e-466cac2a85c1)

2.The compare of 'Temperature VS Relative Humidity' chart in PMV ASHRAE model:
<img width="1038" alt="image" src="https://github.com/user-attachments/assets/b942a76b-7fd5-46a9-90d2-af82ec67d4cb">

 When the red dot outside the neutral sensation region, the corresponding '✘ Does not comply with ASHRAE Standard 55-2023' will be displayed above the show result. 
<img width="1005" alt="image" src="https://github.com/user-attachments/assets/a66b774c-4e0d-413f-905a-b13313aa45ba">

 Can manually select SI or IP unit: 
<img width="912" alt="image" src="https://github.com/user-attachments/assets/3e9640bb-f667-416c-9bce-441b94b9e344">

3.'Temperature VS Relative Humidity' chart in PMV EN model:
![image](https://github.com/user-attachments/assets/51f1b4bb-f672-4be2-aba0-d6c5258c3654)
![image](https://github.com/user-attachments/assets/059bb17f-c960-41b3-8607-68dcaf7b79f2)
![image](https://github.com/user-attachments/assets/c854f739-f70b-4aeb-bd11-b9ac11b24e82)
 when the red dot outside moves to the category IV region, the corresponding '✘ Does not comply with EN-16798' will be displayed above the show result. 
![image](https://github.com/user-attachments/assets/f4092637-929b-41f3-a0be-f9243027ce69)
 Can manually select SI or IP unit: 
![image](https://github.com/user-attachments/assets/d072bddc-9cf4-40b0-8b63-2b1a9d1f3eb5)
 In addition, the value of 'tdp' parameter in annotation is also indeed inconsistent with the value of the original page, I think the original page is wrong: 
![image](https://github.com/user-attachments/assets/f4e8544c-8f34-41ef-9526-fcd19192599f)

4. 'Air speed vs Operature Temperature' chart:
User can select PMV-ASHRAE 55 model and select ‘Air speed vs. Operative temperature’
![image](https://github.com/user-attachments/assets/84a4742b-c16a-404b-a088-07844dde220a)
The input field on the left has the initial values and as the user enters the values, the visualization on the right changes accordingly
<img width="1274" alt="9eac099af5f9e877af6c154e57e3a57" src="https://github.com/user-attachments/assets/475c08a3-3e42-4f37-b84d-2fea31477af8">
The user can also make input of different units by switching units, and the corresponding chart will also change
<img width="1274" alt="fd594c00a83b3baae1e40b742955be9" src="https://github.com/user-attachments/assets/a4d76c10-59ff-40e7-afd9-9486ed8f38e9">

lja996877259 committed:
fix: add air speed vs. temp chart

lja996877259 committed:
fix: add air speed vs. temp chart

CharlieJiangJC committed:
added t_rh pmvashrae/pmven/compare again

Yannnnnnny committed:
Merge pull request #10 from lja996877259/1018

1018

Yannnnnnny committed:
change layout of adaptive chart, invisible lines behind annotation of t_rh chart, stop hover in compare t_rh

Yannnnnnny committed:
change layout of adaptive chart, invisible lines behind annotation of t_rh chart, stop hover in compare t_rh

Yannnnnnny committed:
fix bug when select speed_temp there is an error

Yannnnnnny committed:
fix bug when select speed_temp there is an error

lja996877259 committed:
Merge pull request #1 from Yannnnnnny/10

change some layout details &fix bugs when select speed_temp

lja996877259 committed:
fix: fix the chart of air speed

Yannnnnnny committed:
shop hover in  compare in t_rh

CharlieJiangJC committed:
fix some issues

Yannnnnnny committed:
Merge branch '10' into 1018

Yannnnnnny committed:
Merge pull request #11 from lja996877259/1018

1018

Yannnnnnny committed:
add "set" of pmv ashrae show result, add show result of compare

Yannnnnnny committed:
delete the 'SET' on show result of pmv EN, change the marker of compare chart

lja996877259 committed:
Merge pull request #2 from Yannnnnnny/10

change show result and marker of compare chart

CharlieJiangJC committed:
black code

CharlieJiangJC reviewed (commented):
All code has no syntax errors now.

Yannnnnnny committed:
change color

CharlieJiangJC committed:
Merge pull request #3 from Yannnnnnny/10

change color of

jiadongZhangWorkIn committed:
Update input_environmental_personal.py

After switching to the speed_temp graph, the input box will change from air temperature to operative temperature, and then when switching to another graph, the air temperature will change to operative temperature where it should be. adding this line will fix the bug

jiadongZhangWorkIn reviewed (approved):
Some features have been updated this time and the code looks good

CharlieJiangJC committed:
black again

Mitarashiharuto reviewed (approved):
Some function updates in chart
